### ISSUE TITLE
Vague purpose of dev.md

### ISSUE THREAD

**Issue by Endermanbugzjfc** [SOF3/Capital#46]
dev.md holds >99% of the official API docs (outside Poggit). I remember one day I spent so long trying to find an official API doc of this plugin. But end up giving up because I omitted dev.md after getting told its purpose.

I originally wanted to complain the purpose, "This document is for developers who want to ***contribute to this project***" is misleading. However, while writing this issue, I realized that making (peripheral) plugins based on the Capital API can be seen as a contribution to the "project". Therefore, I say its purpose is vague instead of misleading.
# TL;DR
I do not want to open Poggit to look up the official API doc. But I thought dev.md is for internal dev so I did not read it while looking up on GitHub.

**Comment by SOF3** [comment]
that line is obsolete indeed. it is now for both users and contributors.
